src/invidious/helpers/utils.cr

```diff
@@ -365,3 +365,11 @@ def fetch_random_instance
 
   return filtered_instance_list.sample(1)[0]
 end
+
+def reduce_uri(uri : URI | String, max_length : Int32 = 50, suffix : String = "…") : String
+  str = uri.to_s.sub(/https?:\/\//, "")
```

Comment:
```suggestion
  str = uri.to_s.sub(/^https?:\/\//, "")
```

Reply:
That's minor, but I realized that it's needed for e.g web archive links, which have another `https://` in the URL path.

Reply:
> That's minor, but I realized that it's needed for e.g web archive links, which have another `https://` in the URL path.

It's true I had forgotten that too, I'll change it right away